home-assistant/configuration.yaml

```diff
@@ -6,6 +6,16 @@ input_text:
     name: Alexa Actionable Notification Holder
     max: 255
     initial: '{"text": "This is a test of the alexa actionable notifications skill. Did it work?", "event": "actionable.skill.test"}'
+  alexa_actionable_response:
+    name: Alexa Actionable Response Holder
+    max: 255
+  assist_to_alexa_response:
+    name: Assist Response Holder
+    max: 255
+
+input_boolean:
+  alexa_action_in_progress:
+    name: Alexa Action in Progress

```

Comment:
I have a lot of input_booleans that I created in config.yaml instead of the helper UI, so I know this is correct. It's identical to the input_text you created, but only specifies the name after listting what will be the entity id.

Reply:
Actually its looks correct indeed, I was referencing the code before but I was mistaken. My bad